client/lib/debug/index.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * External dependencies
+ */
+import debug from 'debug';
+
+/**
+ * Internal dependencies
+ */
+import config from 'config';
+
+export default ( ...args ) => {
+	if ( config( 'env_id' ) === 'production' ) {
+		return () => {};
```

Comment:
maybe just use `lodash/noop` or a single noop function to cut down on the number of functions we create?